german/the-black-hack.md

```diff
@@ -113,7 +113,7 @@ Zeit im Spiel wird vor allem auf zwei Arten nachgehalten - **Momente** (Runden)
 
 ## Bewegung und Entfernung
 
-Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Benachbart**, **in der Nähe**, **abseits** und **entfernt**. Während eines Zugs können sich Charaktere irgendwo **in der Nähe** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich **abseits** zu positionieren. Alles jenseits von **abseits** gilt als **entfernt** und würde drei Bewegungen erfordern.
+Anstatt genaue Werte nachzuhalten, benutzt TBH abstrakte Bereiche, mit denen die Entfernung dargestellt wird. **Dicht**, **Nah**, **Weit** und **Fern**. Während eines Zugs können sich Charaktere irgendwo **in der Nähe** bewegen und ihre Handlung jederzeit während der Bewegung durchführen. Sie können ihre Handlung verfallen lassen, um sich **abseits** zu positionieren. Alles jenseits von **abseits** gilt als **entfernt** und würde drei Bewegungen erfordern.
```

Comment:
> ja das wäre eine findige Lösung wenn in Reichweite und in Schussweite auch ungünstig ist, weil in Reichweite auch in Schussweite ist. Vor allem fällt in Schussweite raus wenn es sich nicht um Kampfhandlungen handeln und die Distanz für das Erforschen in der Außenwelt erweitert wird.

Allerdings sehe ich in der 1. Edition nirgends, dass die Distanz irgendwie auf einen anderen Maßstab skaliert wird. Mit Metern nimmt es das Spiel nicht genau, aber von der Größenordnung ist "Close" immer, was ich vor der Nase habe und anfassen kann, "Nearby" immer, was ich mit ein paar Schritten erreichen kann oder was sich im selben Zimmer befindet, "Far-Away", was ich mit einem kurzen Sprint oder einem normalen Fernkampfangriff erreichen kann und "Distant" alles darüber hinaus. Das wird nie auf Dorfentfernungen oder Reisegeschwindigkeiten hochgerechnet.

Und mit "Schussweite" hätte ich auch kein Problem, es geht ja nicht darum, wohin man schießen kann und wohin nicht, sondern darum, ein Gefühl für die gemeinte Entfernung zu geben. Und ca. 20-40 m stimmmen da zumindest bei mir grob mit meiner Vorstellung überein.


"Weit" & "Weit entfernt" als unterschiedliche Kategorien finde ich da z.B. deutlich verwirrender.

Generell ist das hier ein Thema, wo ich schon drüber gegrübelt habe, als ich noch allein an der Übersetzung saß und wo ich für jeden Ansatz höchstens drei passende Begriffe zusammenkriege.

Von der bisherigen Auswahl gefallen mir einigermaßen:
1. In Griffweite - In Reichweite - In Schussweite - In Sichtweite (weil das einigermaßen anschaulich ist, auch ohne die Regeln überhaupt gelesen zu haben)
2. dicht dran - in der Nähe/nahebei - weit(er) weg - in der Ferne (weil das den englischen Begriffen vielleicht am nächsten kommt)